tests/test-cases/Group1-Docker-Commands/1-25-Docker-Port-Map.robot

```diff
@@ -12,9 +12,11 @@ Create container with port mappings
     ${rc}  ${output}=  Run And Return Rc And Output  docker ${params} start webserver
     Should Be Equal As Integers  ${rc}  0
     Should Not Contain  ${output}  Error
-    ${rc}  ${output}=  Run And Return Rc And Output  wget ${vch-ip}:10000 --retry-connrefused --timeout=10 --tries=10
```

Comment:
I read the wget man page, the timeout and retries params do nothing in this case, so let's remove them both.  Not sure about retry-connrefused either, doesn't say how many times it retries - just get rid of that too.  The test does need to wait a bit as there will be time in between start returning and nginx listening on the port, we can use Robot Wait for that.